frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.styled.tsx

```diff
@@ -1,5 +1,80 @@
 import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+import Icon from "metabase/components/Icon";
+import Button from "metabase/core/components/Button";
+import Input from "metabase/core/components/Input/Input";
 
 export const ExpressionWidgetWrapper = styled.div`
   width: 445px;
 `;
+
+export const ExpressionWidgetFieldWrapper = styled.div`
+  padding: 0 1.5rem 1.5rem;
+`;
+
+export const ExpressionWidgetExpressionFieldWrapper = styled.div`
+  padding: 1.5rem 1.5rem 1rem;
+`;
+
+export const ExpressionWidgetFieldTitle = styled.div`
+  margin-bottom: 0.5rem;
+
+  font-weight: 700;
+  font-size: 0.83em;
+  text-transform: uppercase;
+  letter-spacing: 0.06em;
+
+  color: ${color("text-light")};
+`;
+
+export const ExpressionWidgetIconWrapper = styled.span`
+  margin-left: 4px;
+  cursor: help;
+
+  &:hover {
+    color: ${color("text-dark")};
+  }
+`;
+
+export const ExpressionWidgetStyledIcon = styled(Icon)`
+  width: 10px;
+  height: 10px;
+`;
+
+export const ExpressionWidgetInput = styled(Input)`
+  input {
+    border-radius: 4px;
```

Comment:
@mazameli Are we moving towards using `border-radius: 4px` in input boxes? This matches the spec in the [design here](https://www.figma.com/file/68g8S9L4cMesbctGfCySl7/De-crazy-the-expression-editor-function-helper-popover?node-id=0%3A1&t=7VTlgGbjJm0Za6is-0).

But I feel like we still use `8px` in a lot of other places. e.g. dashboard parameter
<img width="379" alt="image" src="https://user-images.githubusercontent.com/1937582/222438282-c4493824-4c95-495f-b68b-231162f2cee8.png">


Found this PR https://github.com/metabase/metabase/pull/23840 which standardizes input box radius from 4px to 8px too.